.github/workflows/deploy.yml

```diff
@@ -27,27 +46,41 @@ jobs:
         password: ${{ secrets.GITHUB_TOKEN }}
     - uses: docker/build-push-action@v4
       with:
-        tags: ${{ env.DOCKER_TAG }}:${{ env.VERSION }},${{ env.DOCKER_TAG }}:latest
+        tags: ${{ env.DOCKER_TAG }}-${{ matrix.build_target.name_suffix}}:${{ env.VERSION }},${{ env.DOCKER_TAG }}-${{ matrix.build_target.name_suffix}}:latest
         pull: true
         platforms: linux/arm64
         push: ${{ github.event_name == 'release' }}
-        file: etc/docker/Dockerfile.triton-jetpack-focal
+        file: ${{ matrix.build_target.dockerfile }}
+        build-args: ${{ matrix.build_target.build_args }}
   module:
     needs: [container]
     strategy:
       matrix:
-        platform: [arm64]
+        platform:
+          - arch: arm64
+            docker_suffix: jetpack5
+            meta_path: meta.jetpack5.json
+          - arch: arm64
+            docker_suffix: jetpack6
+            meta_path: meta.jetpack6.json
+          - arch: arm64
+            docker_suffix: cuda_arm
+            meta_path: meta.cuda.json
+          - arch: x86
+            docker_suffix: cuda_x86
+            meta_path: meta.cuda.json
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
     - name: build
       run: |
-        TAG=$DOCKER_TAG:$VERSION make -f Makefile.module module.tar.gz
+        TAG=$DOCKER_TAG-${{ env.platform.docker_suffix }}:$VERSION make -f Makefile.module module.tar.gz
```

Comment:
$DOCKER_TAG looks like a URL. Put a slash instead of a hyphen before the suffix, or else it'll go to the wrong github account.